packages/dashboard/package.json

```diff
@@ -16,8 +16,6 @@
   "dependencies": {
     "@openmsupply-client/common": "^0.0.1",
     "@openmsupply-client/config": "^0.0.0",
-    "@openmsupply-client/invoices": "^0.0.0",
```

Comment:
The `DashboardService` is still using `InboundShipmentWidget` & `OutboundShipmentWidget` from invoices. Shouldn't the dep be in here?

Reply:
Thanks 😁